airflow/config_templates/default_airflow.cfg

```diff
@@ -1195,6 +1195,11 @@ job_heartbeat_sec = 5
 # how often the scheduler should run (in seconds).
 scheduler_heartbeat_sec = 5
 
+# The frequency (in seconds) at which the LocalTaskJob should send heartbeat signals to the
+# scheduler to notify it's still alive. If this value is set to 0, the heartbeat interval will default
+# to scheduler's `scheduler_zombie_task_threshold` config.
+local_task_job_heartbeat_sec = 0
```

Comment:
Is it reasonable to _want_ the heartbeat interval to be actually zero (i.e. as soon as possible)? I’m wondering if -1 ia better default.

```suggestion
# The frequency (in seconds) at which the LocalTaskJob should send heartbeat signals to the
# scheduler to notify it's still alive. If this value is set to 0, the heartbeat interval will default
# to the value of scheduler_zombie_task_threshold.
local_task_job_heartbeat_sec = 0
```

Reply:
I definitely considered -1 but  if someone sets the value as 0 then we would have to consider that. Since value 0 as a heartbeat value doesn't work, I think 0 makes sense. What do you think?